ballerina/tests/types.bal

```diff
@@ -0,0 +1,2634 @@
+type BooleanRecord1 record {
```

Comment:
Do we need this file for the initial commit? this should go with implementation right?

Reply:
Yes, Since I added the API defs in the `csv_api.bal` file

Reply:
But this should not have those type right? since this is belongs to the tests. Can you verify again?

Reply:
+1